### PR TITLE
Added code of conduct

### PR DESCRIPTION
**Pull request by santacodes**
- Added code of conduct, [contributor covenant](https://www.contributor-covenant.org/) v2.1 as discussed in the last meeting.

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[pybamm@gmail.com](mailto:pybamm@gmail.com).
```
Comment (brosaplanella) [inline CODE-OF-CONDUCT.md:66]:
```suggestion
[pybamm@pybamm.org](mailto:pybamm@pybamm.org).
```

**Review by brosaplanella — APPROVED** [review]
Looks good, just see small suggestion about changing the email.

**Review by agriyakhetarpal — APPROVED** [review]

**Review by brosaplanella — APPROVED** [review]

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
```
Comment (arjxn-py) [inline CODE-OF-CONDUCT.md:1]: This badge should also be updated with the full URL, i.e. [https://www.contributor-covenant.org/version/2/1/code_of_conduct.html](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html), feel free to get this in along #9

Comment (santacodes) [reply CODE-OF-CONDUCT.md:1]: Oh didn't see this review but I have put up another PR for that. #11

Comment (brosaplanella) [reply CODE-OF-CONDUCT.md:1]: Sorry, I didn't see that either and merged it!

Comment (arjxn-py) [reply CODE-OF-CONDUCT.md:1]: No worries, we weren't able to catch that as we don't have link checker configured on this one yet, may be we can look forward to configure it

**Pull request merged**